dataland-frontend/tests/e2e/fixtures/common/CountryFixtures.ts

```diff
@@ -14,7 +14,7 @@ export function generateIso2CountryCode(): string {
  * @returns the randomly generated list of country codes
  */
 export function generateListOfIso2CountryCodes(): string[] {
-  return Array.from({ length: faker.datatype.number({ min: 0, max: 5 }) }, () => {
```

Comment:
Why is this changed?

Reply:
It is changed because the api sorts the entries of this list lexicographically. This leads to failing the editDataset Test because
`["DE", "BE"], != ["BE", "DE"]`, so fixtures need to be sorted as well.

Reply:
If the field before is selected as "Yes", having 0 countries is not allowed

Reply:
ok